packages/jbuilder-x.1.0+beta16/files/jbuilder-x.install

```diff
@@ -0,0 +1,4 @@
+bin: [
+  "jbuilder-x" {"jbuilder-x"}
```

Comment:
This would conflict with a `jbuilder-x` from e.g. `opam-cross-android`.

Reply:
`jbuilder-windows`?

Reply:
Sounds good!